inc/template-functions.php

```diff
@@ -97,3 +97,25 @@ function interconnection_cap_description( $description ) : string {
 	return $description;
 }
 add_filter( 'get_the_archive_description', 'interconnection_cap_description' );
+
+/**
+ * Ensures that the 'pll_duplicate_content' user meta is always set to true.
+ *
+ * @param null|bool  $check      Whether to allow adding/updating metadata of the given type.
+ * @param int        $user_id    User ID.
+ * @param string     $meta_key   Meta key.
+ * @param mixed      $meta_value Meta value.
+ * @return null|bool Null to allow adding/updating metadata, false to prevent it.
+ */
+function filter_pll_duplicate_content( $check, $user_id, $meta_key, $meta_value ) {
+	if ( $meta_key === 'pll_duplicate_content' ) {
+		$meta_value['post'] = true;
+		update_user_meta( $user_id, $meta_key, $meta_value );
```

Comment:
Would this cause an infinite loop? It seems concerning to call a update function inside an action that runs within that update function (`update_user_meta()` calls `update_metadata()` which invokes `update_{meta_type}_meta`).